Bottom: f7d2ba0428817eeaa855ddb332d7f7e51db9326f
Top:    7a63e115cc4adf073292cd3edca568d9910e1a6e
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-10-04 12:01:50 -0700

ucmatose: Remove connect parameter passed into rdma_accept

Pass in NULL for conn_param into rdma_accept to indicate
that the passive side will use the values specified by the
active side.

Signed-off-by: Sean Hefty <sean.hefty@intel.com>


---

diff --git a/examples/cmatose.c b/examples/cmatose.c
index 9ea6777..2b6b2b1 100644
--- a/examples/cmatose.c
+++ b/examples/cmatose.c
@@ -266,7 +266,6 @@ err:
 static int connect_handler(struct rdma_cm_id *cma_id)
 {
 	struct cmatest_node *node;
-	struct rdma_conn_param conn_param;
 	int ret;
 
 	if (test.conn_index == connections) {
@@ -286,10 +285,7 @@ static int connect_handler(struct rdma_cm_id *cma_id)
 	if (ret)
 		goto err2;
 
-	memset(&conn_param, 0, sizeof conn_param);
-	conn_param.responder_resources = 1;
-	conn_param.initiator_depth = 1;
-	ret = rdma_accept(node->cma_id, &conn_param);
+	ret = rdma_accept(node->cma_id, NULL);
 	if (ret) {
 		perror("cmatose: failure accepting");
 		goto err2;
